pg_sparse/README.md

```diff
@@ -0,0 +1,133 @@
+# pg_sparse
+
+## Overview
+
+`pg_sparse` is a PostgreSQL extension that enables similarity search over sparse vectors
+in Postgres using HNSW.
```

Comment:
A quick definition of HNSW will be welcome here by anyone just skimming our repo